lib/stormpath-sdk/resource/organization.rb

```diff
@@ -0,0 +1,11 @@
+class Stormpath::Resource::Organization < Stormpath::Resource::Instance
+  include Stormpath::Resource::CustomDataStorage
+
+  prop_accessor :name, :description, :name_key, :status
```

Comment:
Added the additional fields and tests for them. 

We didn't have created and modified timestamps on other resources so i sticked to that convention. should we start adding them?